src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,23 @@
-export const PostInfo = () => <>Put the post here</>;
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post, userList, commenstList }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <UserInfo
+        user={userList.find(user => user.id === post.userId)}
```

Comment:
```suggestion
export const PostInfo = ({ post, userList, commenstList }) => {
  const user = userList.find(user => user.id === post.userId)

   return (
  <div className="PostInfo">
    <div className="PostInfo__header">
      <h3 className="PostInfo__title">{post.title}</h3>

     {user && <UserInfo
        user={user}
```
make the same with comments and don't forget to import that array